blockchain/state_transition.go

```diff
@@ -135,7 +135,7 @@ type kerror struct {
 
 // NewStateTransition initialises and returns a new state transition object.
 func NewStateTransition(evm *vm.EVM, msg Message) *StateTransition {
-	// before kip71 hardfork, effectiveGasPrice is unitPrice
```

Comment:
Don't we need to change kip71 hardfork to magma hardfork here too?

Reply:
Oh this change is included in https://github.com/klaytn/klaytn/pull/1514. Resolved.